### ISSUE TITLE
error

### ISSUE THREAD

**Issue by terem661** [ZcashFoundation/zecwallet#248]
dobryy den', uvazhayemaya podderzhka uslyshte menya, poymiti situatsiyu i pomogite reshit'. ya obnovil koshelek do 0.9.5 on rabotal, ya tuda perevel monetu. kogda nachel perevodit' s nego. proizoshla oshibka , blokov net soyedineniya i 5 stanovyatsya men'she, skachal 0.8.3 no funktsii reskan i reindeks mne nedostupny. kak mne byt' vse eto na vindovs yest' iskhodyashchaya tranzaktsiya bez podtverzhdeniya
Развернуть
373/5000
Good afternoon, dear support, hear me, understand the situation and help solve. I updated the wallet to 0.9.5 it worked, I transferred the coin there. when I started to translate from him. an error occurred, there are no connection blocks and 5 become smaller, I downloaded 0.8.3 but the rescan and reindex functions are not available to me. how can I be all this on windows there is an outgoing transaction without confirmation

**Comment by terem661** [comment]
Good afternoon, dear support, hear me, understand the situation and help solve. I updated the wallet to 0.9.5 it worked, I transferred the coin there. when I started to translate from him. an error occurred, there are no connection blocks and 5 become smaller, I downloaded 0.8.3 but the rescan and reindex functions are not available to me. how can I be all this on windows there is an outgoing transaction without confirmation
